src/utils/responseTexts.ts

```diff
@@ -0,0 +1,13 @@
+export const deleteStatusMessage = (
+	app: string
+): {
+	success: string;
+	error: string;
+	folderShouldntExist: string;
+	appShouldntExist: string;
+} => ({
+	success: 'Deploy Delete Succeed',
+	error: `Oops! It looks like the application (${app}) hasn't been deployed yet. Please deploy it before you delete it.`,
+	folderShouldntExist: `The folder shouldnt exist after deleting it.`,
```

Comment:
remove these two newly added cases, as they are not required, as the response is for user not the developer.

Remove the logic for folderShouldntExist, appShouldntExist, as all the cases are handled properly.

Also Dont use multiple return statements.

Reply:
Hey ! Thanks for the review I really dont think the cases are being handled properly. for example when scaling it for future there may be cases when the system disconects or any other simultaneous processes may block the current proccesses. I think these cases will be need and thus multiple return statements are necessary

Reply:
I see we shouldnt send it to the user but we need to handle these cases at the backend side IMHO